src/main/app/src/app/core/service/util.service.ts

```diff
@@ -71,12 +73,12 @@ export class UtilService {
     }
 
     setTitle(title: string, params?: {}): void {
-        forkJoin({
-            prefix: this.translate.get('title.prefix', {gemeente: this.readGemeenteNaam()}),
-            title: this.translate.get(title, params)
-        }).subscribe(result => {
-            this.titleService.setTitle(result.prefix + result.title);
-            this.headerTitle.next(result.title);
+        const sources = [];
+        sources['prefix'] = this.translate.get('title.prefix', {gemeente: this.readGemeenteNaam()});
+        sources['title'] = this.translate.get(title, params);
```

Comment:
Een object toekenning op een array?

Reply:
het gebruik van forkjoin met een object is depricated, het gebruik van een array wordt  als alternatief aangeboden